README.md

```diff
@@ -60,6 +60,7 @@ Now you can build the files using one of these commands:
 -   `npm run build` : Build a production version
 -   `npm run dev` : Build a development version
 -   `npm run start` : Build a development version, watch files for changes
+-   `npm run start:hot` : Build a development version in Fast Refresh mode, watch files for changes.
 
```

Comment:
💅  Another option to consider could be `npm run watch`

Reply:
`npm run start` is also running on watch mode. I think it would be better to have the meaning of fast refresh if want to use another script name.